home/screens/ProjectsScreen.js

```diff
@@ -327,6 +328,12 @@ export default class ProjectsScreen extends React.Component {
         <Text style={styles.expoVersionText} onPress={this._copyClientVersionToClipboard}>
           Client version: {Constants.expoVersion}
         </Text>
+        <Text style={styles.supportSdksText}>
+          Supports SDK{Constants.supportedExpoSdks.length === 1 ? "" : "s"} {
+            // Displays the major version numbers of the SDKs in ascending order.
+            Constants.supportedExpoSdks.map(version => semver.major(version)).sort((a, b) => a - b).join(", ")
```

Comment:
```suggestion
            Constants.supportedExpoSdks.map(semver.major).sort().join(", ")
```
This should work too 😉

Reply:
I think even `[9, 10, 11].sort()` (with `int`s) returns `[10, 11, 9]` instead of `[9, 10, 11]`. So we have to use `.sort((a, b) => a - b)`. (See https://stackoverflow.com/q/1063007/2603230)